test/test_runner.sh

```diff
@@ -31,4 +31,4 @@ function execute() {
 TESTS=${@:-.}
 
 # Run the tests.
-execute time bats --tap $TESTS
+execute time bats --jobs $(($(nproc --all)*4)) --tap $TESTS
```

Comment:
I am not sure kata will be able to handle `nproc * 4`  threads. Could this be changed to be able to set the number of jobs by an envar? So that in case of kata, we can adjust this value to something else. Something like: 
```
jobs=${jobs:-$(($(nproc --all)*4))}
execute time bats --jobs "$jobs" --tap $TESTS
```

Reply:
Oh sure I can do that. :+1:  This value mostly comes from my local tests where I saw that one cri-o instance does not utilize the whole CPU.